### PR TITLE
Update to follow latest template

### PR DESCRIPTION
**Pull request by rhagarty**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -136,28 +132,19 @@ DISCOVERY_COLLECTION_ID=<add_discovery_collection>
...
   > To work with a free trial, a small free Discovery environment is created. If you already have a Discovery environment, this will fail. If you are not using Discovery, check for an old service thay you may want to delete. Otherwise use the .env DISCOVERY_ENVIRONMENT_ID to tell the app which environment you want it to use. A collection will be created in this environment using the default configuration.
 
-# License
-[Apache 2.0](LICENSE)
-
 # Privacy Notice
-If using the `Deploy to Bluemix` button some metrics are tracked, the following
-information is sent to a [Deployment Tracker](https://github.com/IBM/metrics-tracker-service) service
+
+If using the Deploy to IBM Cloud button some metrics are tracked, the following information is sent to a [Deployment Tracker](https://github.com/IBM-Bluemix/cf-deployment-tracker-service) service
```
Comment (markstur) [inline README.md]: missed the back ticks around `Deploy to IBM Cloud`

Comment (markstur) [reply README.md]: This change looks like it is going from the new tracker to the one.  Not sure what's going on there.

File: README.md [hunk 2]
```diff
@@ -168,12 +155,30 @@ on each deployment:
...
+This data is collected from the `package.json` file in the sample application and the ``VCAP_APPLICATION`` and ``VCAP_SERVICES`` environment variables in IBM Cloud and other Cloud Foundry platforms. This data is used by IBM to track metrics around deployments of sample applications to IBM Cloud to measure the usefulness of our examples, so that we can continuously improve the content we offer to you. Only deployments of sample applications that include code to ping the Deployment Tracker service will be tracked.
 
 ## Disabling Deployment Tracking
 
-To disable tracking, simply remove ``require("cf-deployment-tracker-client").track();`` and `require("metrics-tracker-client").track();` from the ``app.js`` file in the top level directory.
+To disable tracking, simply remove ``cf_deployment_tracker.track()`` from the ``app.js`` file in the top level directory.
```
Comment (markstur) [inline README.md]: Is this right?

**Review by markstur — CHANGES_REQUESTED** [review]
Missing backticks, but then also not sure about some of the tracker changes.

**Review by stevemar — APPROVED** [review]
you can also remove the cf tracker from package.json

**Pull request merged**